operations/template/storage.tf

```diff
@@ -55,3 +55,45 @@ resource "azurerm_role_assignment" "allow_api_read_write" {
   role_definition_name = "Storage Blob Data Contributor"
   principal_id         = azurerm_linux_web_app.api.identity.0.principal_id
 }
+
+resource "azurerm_storage_account" "automated_storage" {
```

Comment:
Can you set this storage account with encryption with our customer managed key?  You can take inspiration from https://github.com/CDCgov/reportstream-sftp-ingestion/pull/144

Reply:
I'm adding the `azurerm_storage_account_customer_managed_key` for the new container. Do I also need to add a new `azurerm_key_vault_access_policy` unique to this new container? Similar to `azurerm_key_vault_access_policy.allow_storage_storage_account_wrapping`?

Reply:
I just added the changes here: https://github.com/CDCgov/trusted-intermediary/pull/1280/commits/f91a4fa309bf6b0f7f31a998224206df4cc87665